src/TodoInfo.tsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import { useState } from 'react';
+
+type Props = {
+  onEdit: (todo: Todo, editTitle: string) => Promise<void>;
+  onSelect: (todo: Todo) => void;
+  isLoading: boolean;
+  todo: Todo;
+  onDelete?: (id: number) => Promise<void>;
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  onEdit,
+  onSelect,
+  isLoading,
+  todo,
+  onDelete = () => {},
+}) => {
+  const [isEdited, setIsEdited] = useState(false);
+  const [editTitle, setEditTitle] = useState(todo.title);
+
+  const HandleEdit = () => {
```

Comment:
The same as above, fix it everywhere
```suggestion
  const handleEdit = () => {
```